pkg/booleanpolicy/augmentedobjs/construct.go

```diff
@@ -143,6 +143,33 @@ func ConstructNodeWithFileAccess(node *storage.Node, fileAccess *storage.FileAcc
 	return nodeObj, nil
 }
 
+func ConstructDeploymentWithFileAccess(
+	deployment *storage.Deployment,
+	images []*storage.Image,
+	applied *NetworkPoliciesApplied,
+	fileAccess *storage.FileAccess,
+) (*pathutil.AugmentedObj, error) {
+	obj, err := ConstructDeployment(deployment, images, applied)
+	if err != nil {
+		return nil, err
+	}
+
+	augmentedAccess, err := ConstructFileAccess(fileAccess)
+	if err != nil {
+		return nil, err
+	}
+
+	err = obj.AddAugmentedObjAt(augmentedAccess, pathutil.FieldStep(fileAccessKey))
+	if err != nil {
+		return nil, err
+	}
+	return obj, nil
+}
+
+func ConstructFileAccess(fileAccess *storage.FileAccess) (*pathutil.AugmentedObj, error) {
```

Comment:
This does not need an error return right? That would also simplify the calling function where this func is called.